.eslintignore

```diff
@@ -0,0 +1,2 @@
+**/test/fixtures/**/node_modules/**
+**/test/work/*
```

Comment:
I didn't really want to add another dotfile but it seems that eslint has technical reasons to keep this one: https://github.com/eslint/eslint/issues/4261

*(The "cascading" reason is a bit silly IMO though. The execution one is quite reasonable.)*